lib/generators/services/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+
+services:
+  <%= config[:service] %>:
+    container_name: <%= config[:service] %>
+    build: "./"
+    platform: "linux/x86_64"
```

Comment:
You may want to make this an optional thing to generate. Most of the services out there (that I have encountered) have binaries available for both x86_64 as well as arm64. The big omission is MySQL.

Reply:
@joshpennington I've converted that to an optional param now.